src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1582,15 +1582,19 @@ NOTE: Please follow the graphiql endpoint: `http://localhost:8080/catalog/graphi
 == Usage of Catalog from GUI
 
 The GUI interaction with the Catalog can be done in two places: Studio portal and Catalog Portal.
-The portals follow the concepts of the Catalog: Workflows store inside buckets, the Workflow can have several revisions, each Workflow have specific description.
-The view of Workflows from Catalog in Studio and from Catalog Portal follow the same view logic. The page is separated in 3 sections.
-The approach is selecting the needed information from left to right. First user should choose the required bucket, then the user can observe all the Workflows in the selected bucket. When the user select the Workflow it will be shown all existing revisions of the Workflow. By selecting a revision you will be able to see the the description about the workflow revision.
+The portals follow the concepts of the Catalog: Workflows are stored inside buckets, a workflow has some metadata and can have several revisions.
+
+The Catalog view in the Studio or in the Catalog Portal allow to browse all the workflows in the Catalog grouped by buckets and to list all the revisions of a workflow along with their commit message.
 
 === Studio
-Inside the Studio portal there is a `Catalog` menu from which a user can directly interact with the Catalog storage.
+Inside the Studio portal there is a `Catalog` menu from which a user can directly interact with the Catalog to import or publish Workflows.
 
 image::studio_catalog_menu.png[align=center]
 
+Besides, the Palette of the Studio lists the user's favourite buckets to allow easy import of workflows from the Catalog with a simple Drag & Drop.
```

Comment:
I'd use "Additionally" or "In addition"  instead. "Besides" is not commonly at the beginning of a paragraph.